silkworm/infra/grpc/client/unary_rpc.hpp

```diff
@@ -52,29 +52,30 @@ class UnaryRpc<Async> {
   private:
     template <typename Dispatcher>
     struct Call {
-        UnaryRpc& self_;
-        const Request& request_;
-        [[no_unique_address]] Dispatcher dispatcher_;
+        UnaryRpc& self;
+        const Request& request;
+        [[no_unique_address]] Dispatcher dispatcher;
```

Comment:
these should be private class members IMO

Reply:
done